text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+# RFC-0000: Lowering Deposit Requirements on Polkadot Asset Hub
+
+|                 |                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                                                    |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot Asset Hub, making it more accessible and affordable for artists. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu)    |
+
+## Summary
+
+This RFC proposes a change to the current deposit requirements on the Polkadot and Kusama Asset Hub for creating NFT collections. The objective is to lower the barrier to entry for artists, fostering a more inclusive and vibrant ecosystem, while maintaining network integrity and preventing spam.
+
+## Motivation
+
+The current deposit of 10 DOT for collection creation on the Polkadot Asset Hub presents a significant financial barrier for many artists. By lowering the deposit requirements, we aim to encourage more artists to participate in the Polkadot NFT ecosystem, thereby enriching the diversity and vibrancy of the community and its offerings.
+
+## Stakeholders
+
+- **Artists**: Primary beneficiaries of the proposed change, particularly those who found the current deposit requirements prohibitive.
+- **KodaDot**: As the facilitator of artists' relations, KodaDot has a vested interest in making the platform more accessible.
+- **Polkadot Community**: Stands to benefit from an influx of artists, creators and diverse NFT collections, enhancing the overall ecosystem.
```

Comment:
I think the community is also interested in the security of the network.

Reply:
Hmm, since this is talking more about the benefits for Stakeholders, would it be okay if I better highlight this point in the drawbacks?

Reply:
But the Polkadot Community also benefits from a secure network. Otherwise if the state gets bloated like on Moonbeam then it is not good.